data/data_download/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM osgeo/gdal:ubuntu-full-3.2.1
+LABEL maintainer="hello@vizzuality.com"
+ENV NODE_OPTIONS=--max_old_space_size=16384
+RUN apt-get update \
+    && curl -sL https://deb.nodesource.com/setup_13.x | sh - \
```

Comment:
Please split these lines into multiple RUN commands.

Run all apt-get install in a single command

Also: any reason why you are using a dead version of nodejs? if not, please use the same major version as the API/FE apps